components/angular-tomitribe-select/src/tomitribe-select.ts

```diff
@@ -151,6 +153,26 @@ module tomitribe_select {
         }
     }
 
+    function tribeSelectLoadOnOpen() {
+        return {
+            restrict: 'A',
+            require: 'uiSelect',
+            replace: false,
+            link: link,
+        };
+
+        function link(scope, element, attrs, uiSelectCtrl) {
+            scope.$on('uis:activate', ()=> {
+                //Check if we have the function defined  in tribe-select-fetch-on-open="function". Otherwise, check if the have the refresh attribute defined in ui-select
+                let refreshFn = attrs['tribeSelectFetchOnOpen'].indexOf('tribe-select-fetch-on-open') !== 0 ? attrs['tribeSelectFetchOnOpen'] : element.find('.ui-select-dropdown').attr('refresh');
```

Comment:
This is just wrong, please fix indexOf extravaganza, 
also if possible do not use element.find you can:
 - use attribute on ui-select-choices
 - require both uiSelectCtrl and uiSelectChoicesCtrl require: ['uiSelect', 'uiSelectChoices']
 - have uiSelectCtrl.refresh and refresh function from attrs there (attrs['refresh'] without tribeSelectFetchOnOpen function)